babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    plugins: [
+      'annotate-pure-calls',
+      'dev-expression',
```

Comment:
This appears to be unused/unnecessary in react-apollo, or am I missing something?

Reply:
I think you're right. There might be an opportunity to make use of `__DEV__` within `react-apollo`, but right now this only helps because some of `react-apollo`'s (React) dependencies use the `__DEV__` expression, and the `dist/bundlesize.js` file generated by the `bundlesize` check includes those dependencies.

Reply:
Also this is not even the big savior, the big savior is the pure calls. I can just remove dev expression and still save 4KB